src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -100,6 +100,28 @@ end
 
 Base.:*(Aadj::Adjoint{T,<:LazyHcat}, B::AbstractMatrix) where {T} = Aadj * LazyHcat(B)
 
+# Special case of Hermitian result: can only actively compute the block upper diagonal
+@views function mul_hermi(Aadj::Adjoint{T,<:LazyHcat}, B::LazyHcat) where {T}
```

Comment:
Aadj -> A

Reply:
(mul does A*B, not Aadj*B)

Reply:
True. To be fair, I am copying the convention from the original general function https://github.com/JuliaMolSim/DFTK.jl/blob/dc62a84d0ba7f01c6b8307e4c0b26c38e78cdeae/src/eigen/lobpcg_hyper_impl.jl#L83 I guess I could change it there too

Reply:
Yes that'd be good!